src/settings/ubuntuSettings.json

```diff
@@ -121,7 +127,7 @@
             "fileUris": "[variables('ubuntuScripts')]"
           },
           "protectedSettings": {
-            "commandToExecute": "[concat('bash elasticsearch-install.sh -x', variables('commonShortOpts'), variables('commonInstallParams'))]"
+            "commandToExecute": "[concat('bash elasticsearch-install.sh -x', variables('commonShortOpts'), variables('commonInstallParams'), ' -D ', variables('quote'), parameters('dataDogAPIKey'), variables('quote'))]"
```

Comment:
@tyrotoxin Here is where we use the Parameter by passing it in using the new `-D` option on the `elasticsearch-install.sh` script. We wrap the key in quotes because I think it removes them when passed into the ARM Template (at least thats what I am seeing done with other parameters and variables)